### DISCUSSION TITLE
How to use plug-ins to encrypt some articles

### DISCUSSION THREAD

**Discussion by nanbowanya** [vuejs/vitepress#2992] [Q&A]
### Is your feature request related to a problem? Please describe.

Vitepress is really great, but I would love to add the ability to encrypt access to specific articles.

### Describe the solution you'd like

I tried asking ai, and the following is the reply ai gave me.
![CleanShot 2023-09-20 at 11 51 23@2x](https://github.com/vuejs/vitepress/assets/20452341/a5937173-d7a3-44f4-b0a4-280917c30034)
But I don’t know how to add and use the plug-in in vitepress. Can you tell me how I can use the function of this plug-in. Encrypt the specified document.

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Read the [Contributing Guidelines](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

**Comment by brc-dd** [comment]
VuePress plugins are not supported. You'll need to write a plugin yourself or find one that supports vitepress.
